src/styles/utils/_animations.scss

```diff
@@ -0,0 +1,31 @@
+@mixin animate($animation-name) {
+  animation-name: $animation-name;
+  animation-duration: 0.3s;
+  animation-fill-mode: forwards;
+  animation-timing-function: linear;
+}
+
+@keyframes hover-card {
+  100% {
```

Comment:
The animation 'hover-card' only has one keyframe. It would be better to include initial state, usually at 0%.

```
0% {
            transform: scale(1.0);
        }
```